src/os/freertos/etcpal/os_queue.c

```diff
@@ -106,6 +106,11 @@ bool etcpal_queue_receive_from_isr(etcpal_queue_t* id, void* data)
   return false;
 }
 
+bool etcpal_queue_reset(etcpal_queue_t* id)
+{
+  return xQueueReset(*id);

```

Comment:
Add null checks for `id` in all these functions and return a default/false value if null (see other already defined functions)